.gitignore

```diff
@@ -23,5 +24,4 @@ dist-ssr
 *.sln
 *.sw?
 
-*storybook.log
```

Comment:
There was no mention in description, but these logs no longer get generated or why it is removed also?

Reply:
Fixed back.